### ISSUE TITLE
Error when using AppImage

### ISSUE THREAD

**Issue by Ivan228-s** [yui0/berry-os#2]
> sudo ~/Загрузки/berry-os-*-x86_64.AppImage

```
/tmp/.mount_berry-PalYFa/AppRun: 13: pushd: not found
Archive:  /tmp/.mount_berry-PalYFa/data.zip
error:  cannot create data.vmdk
        Read-only file system
/tmp/.mount_berry-PalYFa/AppRun: 15: popd: not found
qemu-system-x86_64: -drive index=1,if=virtio,file=/root/.local/share/berry-os/data.vmdk,format=vmdk: Could not open '/root/.local/share/berry-os/data.vmdk': No such file or directory
```

**Comment by yui0** [comment]
Hello.

I tried berry-os-12-20230222.glibc2.34-x86_64.AppImage.
But it worked normally.

This application uses the ~/.local/share/berry-os/ directory.
The code is this.
mkdir -p ~/.local/share/berry-os/

So check your directory.

**Comment by fatih-asici** [comment]
Same problem with Debian. Its /bin/sh does not support pushd/popd command. Before the first run, I gave this command: 

`sudo mount --bind /bin/bash /bin/sh`

and it works.

**Comment by yui0** [comment]
Thank you for your information.
I'll fix this problem.

**Comment by SMNFXCN** [comment]
While launching the latest berry-os-9 (2023-02-16) appimage, I get this error on my Steam Deck:
`(deck@steamdeck Applications)$ ./berry-os-9.glibc2.34-x86_64.AppImage
Failed to open module: /usr/lib/libtracker-sparql-3.0.so.0: undefined symbol: g_task_set_static_name
qemu: module ui-ui-gtk not found, do you want to install qemu-system-gui package?
qemu-system-x86_64: Display 'gtk' is not available.
(1)(deck@steamdeck Applications)$`
Any fix for this? I'd rather not install packages that will be wiped after an update.

**Comment by elpendor** [comment]
> While launching the latest berry-os-9 (2023-02-16) appimage, I get this error on my Steam Deck: `(deck@steamdeck Applications)$ ./berry-os-9.glibc2.34-x86_64.AppImage Failed to open module: /usr/lib/libtracker-sparql-3.0.so.0: undefined symbol: g_task_set_static_name qemu: module ui-ui-gtk not found, do you want to install qemu-system-gui package? qemu-system-x86_64: Display 'gtk' is not available. (1)(deck@steamdeck Applications)$` Any fix for this? I'd rather not install packages that will be wiped after an update.

Same here

**Comment by boubiro** [comment]
/tmp/.mount_berry-aYNrpV/AppRun: line 17: /sys/kernel/mm/ksm/run: 
Permission denied Failed to open module: /lib64/libtracker-sparql-3.0.so.0: 
undefined symbol: g_task_set_static_name qemu:
module ui-ui-gtk not found, do you want to install qemu-system-gui package? 
qemu-system-x86_64: Display 'gtk' is not available.

Hello, I also have the same error, although it worked well before, did you have a fix?

**Comment by InternetRebelAKAQuachHungPhong** [comment]
Could this be because this appimage is extremely optimized for Fedora/Red Hat Linux, or it doesn't have dependencies within it?

**Comment by 4180** [comment]
> Could this be because this appimage is extremely optimized for Fedora/Red Hat Linux, or it doesn't have dependencies within it?

Yes, yes it is, it's heavily optimized for Fedora, so i decided to use distrobox, and it kinda... worked, it was able to do the data.zip, but the 

```
module ui-ui-gtk not found, do you want to install qemu-system-gui package?
qemu-system-x86_64: Display 'gtk' is not available.
```

remained...

the appimage is practically just a executable which unzips the data folder and it's other components...

I'm not a developer, but if i can, I'll try to configure the project to work for debian-based distros.
